src/main/java/ch/njol/skript/util/PotionEffectUtils.java

```diff
@@ -76,6 +87,17 @@ public static PotionEffectType parseType(final String s) {
 		return types.get(s.toLowerCase());
 	}
 	
+	// This is a stupid bandaid to fix comparison issues when converting potion datas
```

Comment:
What are the comparison issues?

Reply:
Its in the note of this PR, should I add it there too?

Reply:
might not be a bad idea